### DISCUSSION TITLE
Can't find Tennis Doubles Matches on the API

### DISCUSSION THREAD

**Discussion by kloupa** [cloudbet/docs#76] [Q&A]
Hello, I try get the matches information for all tennis matches with the API.
I got no problem getting information for Singles matches, but when I get the list of all competitions available there are no Doubles for tennis.
I can clearly see on the website app that there are many matches for double in tennis so I get the ID of the match on the website and try get the information via the API but I got 404 error.
It looks like the API got no information about doubles match in tennis,
Did someone got the same problem, did I need to activate something into my account or something ?

**Comment by kgravenreuth** [comment]
Hi @kloupa ,

Apologies for the delay in reply. 

Could you please mention the specific Tennis doubles competition where you are seeing issues together with a sample curl request to the Cloudbet API (with your API Key removed)?

We do offer Tennis doubles in general via our API, while there might be some specific competitions which could be offered on our website but not via our API at our discretion.

Regards,
Klaus Gravenreuth
Product & Engineering @ Cloudbet
